### PR TITLE
docs: add README for E2E test scripts

### PR DESCRIPTION
**Pull request by AllanFly120**
The E2E test requires Chrome and Firefox exists in the environment. It's hard to provide a guideline to install browsers to all possible platforms, expecially AL2, which doesn't support installing Firefox from a package manager(yum). This README uses AWS standard docker image, which comes with all the resources needed for E2E test.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Prerequisites
+
+1.  You have a CloudFormation stack called `SdkReleaseV3IntegTestResourcesStack` in the AWS account you used to run the
+    integration test. If not you can deploy the stack with:
+
+    ```console
+    aws cloudformation deploy --stack-name SdkReleaseV3IntegTestResourcesStack --template-file ./IntegTestResourcesStack.template.json --capabilities CAPABILITY_IAM
+    ```
+
+1.  You have `ubuntu/standard/4.0` docker image built. If not, please follow [the guide to build the image](https://github.com/aws/aws-codebuild-docker-images#how-to-build-docker-images).
```
Comment (trivikr) [inline tests/e2e/README.md:10]: Installation of `aws/codebuild/standard:4.0` fails on AL2

Created GitHub issue at https://github.com/aws/aws-codebuild-docker-images/issues/375

Comment (trivikr) [reply tests/e2e/README.md:10]: The issue was fixed by an alternative provided in https://github.com/aws/aws-codebuild-docker-images/issues/375#issuecomment-668946363

File: tests/e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+1. Inside the container, configure the credentials to be the one that has permission to the `SdkReleaseV3IntegTestResourcesStack`
+   stack. You can configure the credentials following [this guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+
+1. Change directory to the SDK directory, make sure the clients that contains end-to-end test suites has been built to
+   JS files.
+
+1. Run the end-to-end test suite from the SDK root:
+
+   ```console
+   yarn test:e2e
```
Comment (trivikr) [inline tests/e2e/README.md:43]: The command fails on my docker image with exit code 1 and no error

```console
$  yarn test:e2e
yarn run v1.22.4
...
...
$ node ./tests/e2e/index.js
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Comment (AllanFly120) [reply tests/e2e/README.md:44]: I think the reason is that the `region` is incorrect. I have updated it in 20855e3

**Pull request merged**